### PR TITLE
Documentation

### LINKED ISSUE #136: Provide a global SDK documentation
- [ ] Refactor the main Readme
- [ ] Add the its status new project
- [ ] Add the its info new project
- [ ] Add the its interqueuemanager new project
- [ ] Add the its vehicle new project
- [ ] Update the its client for the Python part
- [ ] Update the its client for the Rust part
- [ ] Add the its client for the Java part

GIVEN a developer using the SDK gathered the main README of the repository
WHEN they click on the relevant package for them (Schema, Rust, Java or Python)
THEN a dedicated README describing the specific API will be available

### PR DESCRIPTION
**Pull request by nbuffon**
Rewrite the main README.md providing:

- The main section telling about what's going on here, presenting the features
- A dedicated section for the JSON schemas
- A quick presentation of the languages supported

Closes #136

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,56 @@ its-client
...
 
-This Intelligent Transportation Systems (ITS) [MQTT][1] client based on
-the [JSON][2] [ETSI][3] specification transcription provides a ready to connect project for the mobility
-(connected and autonomous vehicles, road side units, vulnerable road users,...).
+This repository provides reference implementation of message exchanging for IoT3 over [MQTT][1] (v5)  
+On top of this it also provides a proposal of implementation for V2X messages in [JSON][2] based on [ETSI][3]'s
+Intelligent Transportation Systems (ITS)
 
-Let's connect your device or application to our Intelligent Transport Systems (ITS) platform!
+For both of these use cases telemetry is both automatically done and abstracted to send custom traces
```
Comment (ymorin-orange) [inline README.md:10]: Spurious double trailing space.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,56 @@ its-client
...
+The link between the span of a message publish and the span of its reception is automatically made 
+by propagating the [W3C Trace Context][12] using MQTTv5 properties
+
+Sent traces include the following parameters:
+
+- `service_name` client app name (configurable)
+- `span.status` Error if anything went wrong, Unset otherwise
+- `span.name` IoT3 Core MQTT Message
+- `span.kind`
+  - `consumer` when receiving a message
```
Comment (ymorin-orange) [inline README.md:47]: Spurious trailing space.

File: README.md [hunk 3]
```diff
@@ -48,14 +48,31 @@ Sent traces include the following parameters:
...
     - `iot3.core.mqtt.payload_size` 
-    - `iot3.core.sdk_language`
-
-Packages
---------
-
-### JSON Schema
-
-[ETSI.org][3] proposal of implementation using the [JSON][2] language (instead of ASN.1 UPER by default).
+    - `iot3.core.sdk_language`
```
Comment (ymorin-orange) [inline README.md:52]: Unfortunate spurious trailing space.

File: README.md [hunk 4]
```diff
@@ -2,18 +2,56 @@ its-client
...
+| Language | Traces | Logs | Metrics |
+|----------|--------|------|---------|
+| Rust     | ✓      |      |         |
+| Python   | ✓      |      |         |
+| Java     | ✓      |      |         |
+
+#### Traces
+
+Depending on client's choice or implementation MQTT message publishing and reception might be automatically traced  
+The link between the span of a message publish and the span of its reception is automatically made
```
Comment (ymorin-orange) [inline README.md:38]: Spurious trailing space.

File: README.md [hunk 5]
```diff
@@ -2,18 +2,56 @@ its-client
...
+
+| Language | Traces | Logs | Metrics |
+|----------|--------|------|---------|
+| Rust     | ✓      |      |         |
+| Python   | ✓      |      |         |
+| Java     | ✓      |      |         |
+
+#### Traces
+
+Depending on client's choice or implementation MQTT message publishing and reception might be automatically traced
```
Comment (ymorin-orange) [inline README.md:37]: Missing period at end of sentence.

File: README.md [hunk 6]
```diff
@@ -48,14 +48,31 @@ Sent traces include the following parameters:
...
+
+As mentioned you can find in the `schemas` directory a proposal of implementation using the [JSON][2] language
+(instead of ASN.1 UPER by default) of the following [ETSI.org][3] messages:
+- CooperativeAwarenessMessage (CAM)
+- CollectivePerceptionMessage (CPM)
+- DecentralizedEmergencyNotificationMessage (DENM)
+
+But also schemas of custom messages for V2X:
+- Information (information about service instance)
+- Status (information about a device)
```
Comment (ymorin-orange) [inline README.md:78]: The status message is not about _information_, it is about providing a (kind of) health condition of the instance (which might be a device). So I'd describe it as: _status of an instance_.

File: README.md [hunk 7]
```diff
@@ -74,6 +74,11 @@ they are using the following versions:_
 Languages
 ---------
 
+The features presented above are hereby provided in several languages in an SDK-like form
+
+Choice has been made to respect each language paradigm over providing a unique implementation
+it might therefore differ depending on the language
```
Comment (ymorin-orange) [inline README.md:94]: Missing periods at end of sentences.

**Review by ymorin-orange — CHANGES_REQUESTED** [review]
Minor typos here and there, otherwise good job! 👍

**Review by fredOG-2A1 — APPROVED** [review]

**Review by fredOG-2A1 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nbuffon) [reply README.md:10]: That's one markdown way to line break, but it appears it should also be possible using a backslash, I'll try it (I agree trailing spaces are not the best way to line break)

Comment (nbuffon) [reply README.md:10]: Oh and there is nothing to line break here in fact...

Comment (ymorin-orange) [reply README.md]: Yes, that was the issue: there is no reason to force a line break here as this is already the end of a paragraph.

Comment (ymorin-orange) [reply README.md]: Also, to be clear: using trailing spaces to force a line break was perfectly acceptable (I see you also changed the other s to use a backslash `\`, which is not what this comment was about, sorry for the confusion...)

**Review by mathieu1fb — COMMENTED** [review]
Excellent! Just a few comas to add here and there.

**Comment by nbuffon** [comment]
@mathieu1fb I pushed changes with following your comments

**Review by mathieu1fb — APPROVED** [review]
All good for me following the latest updates.

**Pull request merged**